extensions/interactions/MultipleChoiceInput/directives/oppia-interactive-multiple-choice-input.component.ts

```diff
@@ -89,6 +89,8 @@ export class InteractiveMultipleChoiceInputComponent implements OnInit {
       }
       return choices;
     };
+    // If choices need not be shuffled, order the choices based on
+    // originalIndex property in choicesWithIndex.
```

Comment:
Sorry I don't quite follow, also I'm still not sure why we need to order them.

Reply:
Oh Sorry, let me try again: In the customisation args, we have an option to maintain the order of choices or shuffle them. If the creator decides to maintain the order, we should order the choices based on the "originalIndex" so that that, in the learner view, the choices are always in the same order.

@vojtechjelinek  please let me know if this is clearer, I'll edit the comment. Thanks!

Reply:
```suggestion
    // If choices need be shuffled, shuffle them, otherwise order the choices based on
    // their original index.
```

Reply:
done, thanks!